swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+import uuid
+
+
 class Item:
-    pass
\ No newline at end of file
+   
+    def __init__(self, id=None, condition=0):
+        if id is None:
+            self.id = uuid.uuid4().int
+        else:
+            self.id = id
+        self.condition = condition
+  
+    def get_category(self):
+        return self.__class__.__name__
+
+    def __str__(self):
+        return f"An object of type Item with id {self.id}."
```

Comment:
Rather than hard code the type of class name this is(ie, `type Item`), how might we refactor this? That way any child classes can use the super method `__str__` rather than making their own, which is also hard coded? Maybe we could incorporate `get_category`